lib/src/view/settings/board_settings_screen.dart

```diff
@@ -263,6 +287,44 @@ class PieceShiftMethodSettingsScreen extends ConsumerWidget {
   }
 }
 
+class CastlingMethodSettingsScreen extends ConsumerWidget {
+  const CastlingMethodSettingsScreen({super.key});
+
+  static Route<dynamic> buildRoute(BuildContext context) {
+    return buildScreenRoute(
+      context,
+      screen: const CastlingMethodSettingsScreen(),
+      title: 'Castling method',
```

Comment:
Use the translated string: preferencesCastleByMovingTheKingTwoSquaresOrOntoTheRook